src/tests/Dialog.test.tsx

```diff
@@ -0,0 +1,41 @@
+import { render, fireEvent } from '@testing-library/react'
+import { Dialog } from '../components/Dialog/Dialog.tsx'
+
+describe('Dialog', () => {
+    const onCloseMock = jest.fn()
+
+    const props = {
+        title: 'Test Dialog',
+        children: <div>Test Children</div>,
+        onClose: onCloseMock,
+    }
+
+    it('renders without crashing', () => {
+        const { container } = render(
+            <Dialog {...props}>{props.children}</Dialog>,
```

Comment:
You are reusing the same exact children object in all the tests. This approach is prone to bugs, please use a setup function to make the tests independent from each other.